scripts/ios-build-libraries.sh

```diff
@@ -8,15 +8,27 @@ set +v
 # The main purpose of this script is build and prepare PA2 "fat" libraries for
 # library distribution. Typically, this script is used for CocoaPods integration.
 # 
-# The result of the build process is one multi-architecture framework library 
-# (also called as "fat") with all supported microprocessor architectures in 
-# one file.
-# 
+# The result of the build process is:
+#    libPowerAuthCore.a:
+#      multi-architecture static library (also called as "fat") with all 
+#      core functionality of PowerAuth2 SDK. The library contains all C++
+#      code, plus thin ObjC wrapper written on top of that codes.
+#
+#    SDK sources:
+#      all SDK higl level source codes are copied to destination directory.
```

Comment:
Typo in comment - "high" vs. "higl"

Reply:
I removed that files just now. That was crap from xcode...

Reply:
Fixed that typo.